content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -1,258 +1,148 @@
 {{< /note >}}
 
-The PhysX Collider component requires the [PhysX](/docs/user-guide/gems/reference/physics/nvidia/physx/) gem enabled in your project.
-
-For more information, see [Simulating physics behavior with the PhysX system](/docs/user-guide/interactivity/physics/nvidia-physx/).
-
-## PhysX Collider properties
-
-![PhysX Collider component interface.](/images/user-guide/component/physx/physx/ui-physx-collider-A.png)
-
-**Collision Layer**
-The collision layer that's assigned to the collider. For more information, see [Collision Layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).
-
-**Collides With**
-The collision group containing the layers that this collider collides with. For more information, see [Collision Groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/).
-
-**Trigger**
-Set this collider as a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap between colliders is sufficient.
-Triangle meshes are not supported as triggers.
+## Provider
 
-**Simulated**
-When enabled, this shape collider will be part of the physics simulation.
+[PhysX Gem](/docs/user-guide/gems/reference/physics/nvidia/physx/)
 
-**In Scene Queries**
-When enabled, this shape collider can be queried for raycasts, shapecasts and overlap.
+## Properties 
 
-**Offset**
-Local offset position of the collider, relative to the entity.
+![PhysX Collider component interface.](/images/user-guide/components/reference/physx/physx-collider-ui-01.png)
 
-**Rotation**
-Local rotation of the collider about the **Offset** of the PhysX collider component.
+### Base properties
 
-**Library (Physics Materials)**
-The physics material library of the project.
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Collision Layer** | Assigns the collider to a collision layer. Collision layers can be used to restrict physical interactions between PhysX objects. | Any collision layer defined the project [Collision Layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).  | `Default` |
+| **Collides With** | Assigns the collider to a collision group. Collision groups contain the collision layers that this collider can collide with. | Any collision group defined the project [Collision Groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/). | `All` |
+| **Trigger** | If enabled, this collider functions as a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap test between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `Disabled` |
```

Comment:
```suggestion
| **Trigger** | If enabled, this collider functions as a trigger. A trigger checks if other colliders are overlapping with this collider. It does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap test between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `Disabled` |
```

- I suggest expanding on what "overlap test" means. 
- If you accept the point above, consider splitting up the sentence.

Reply:
I'm going to make a small change to the suggestion. I think splitting the sentence is good, but the most critical bit of information is that overlap test are very quick.

```suggestion
| **Trigger** | If enabled, this collider functions as a trigger. A trigger performs quick overlap tests with other colliders. Triggrers don't apply forces or return contact point information. Use this to speed up PhysX computations in which a simple overlap test between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `Disabled` |
```